keps/sig-node/1029-ephemeral-storage-quotas/kep.yaml

```diff
@@ -16,8 +16,8 @@ editor: TBD
 creation-date: 2018-09-06
 last-updated: 2023-08-31
 status: implementable
-latest-milestone: "1.29"
```

Comment:
This document also needs update with the latest template https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml it's way out of date :sweat_smile:

Reply:
Done